index.html

```diff
@@ -291,6 +291,16 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <div class="contact-container">
+      <h2 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h2>
+      <form action="https://formspree.io/f/xvodkydo" method="POST">
+        <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required>
+        <input type="email" id="email" name="_replyto" placeholder="Email address">
```

Comment:
- [ ] [required] I would suggest you validate this email as well, so that when the form is submitted, the form handler will know how to reach you

Reply:
Done